src/composables/use-seekable.ts

```diff
@@ -0,0 +1,100 @@
+import { computed, ToRefs, ref } from '@nuxtjs/composition-api'
+
+import { useI18n } from '~/composables/use-i18n'
+import { keycodes } from '~/constants/key-codes'
+
+export interface UseSeekableOptions
+  extends ToRefs<{
+    duration: number
+    currentTime: number
+    isReady: boolean
+  }> {
+  onSeek: (frac: number) => void
+  onTogglePlayback: () => void
+}
+
+export const useSeekable = ({
+  duration,
+  currentTime,
+  isReady,
+  onSeek,
+  onTogglePlayback,
+}: UseSeekableOptions) => {
+  const i18n = useI18n()
+
+  const attributes = computed(() => ({
+    'aria-valuemax': duration.value,
```

Comment:
I think we may need to add [`'role': 'slider'`](https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-seek.html) :thinking:

Reply:
`'role': 'slider'` is currently only added when the Waveform is interactive. When it's not (on the search page), then we either need to remove these aria-attributes, or to move the `'role': 'slider'` here so that it's always applied. Can we have slider role inside of an aria-disabled element, though? I'm confused

Reply:
> Can we have slider role inside of an aria-disabled element, though

I think so, but it should not be hard to test. Can you check if VO ignores the role of elements inside an aria-disabled container, for example?